### PR TITLE
Fix bugs and code smells from sonarqube

### PR DESCRIPTION
**Pull request by maexled**

### CODE REVIEW

**Review by madebyTimo — APPROVED** [review]

**Review by delvh — APPROVED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import 'vue-toastification/dist/index.css';
 
 import { hasToken } from './ts/login-rest-client';
 
-router.beforeEach((to, from, next) => {
+router.beforeEach((to, _from, next) => {
```
[the remark this answers is not in the document]
Comment (delvh) [reply src/main.ts:15]: Unfortunately, I can't prove it right now as it is down (once again).

**Comment by delvh** [comment]
Sorry about that, I just noticed that SonarQube indeed wants it to only start with an underscore, not to only be an underscore.

**Comment by maexled** [comment]
Please recheck if I merged correctly.

**Review by delvh — APPROVED** [review]
Seems to be the case.

**Review by Gr33ndev — APPROVED** [review]

**Pull request merged**